gradio/event_queue.py

```diff
@@ -14,8 +14,12 @@
 
 class Estimation(BaseModel):
     msg: Optional[str] = "estimation"
+    rank: Optional[int] = -1  # waiting duration for the xth rank:
```

Comment:
I think we can remove these comments. The purpose of the variables should be evident from their names, and the logic to calculate them can be found in the code itself.

Reply:
Added because the meaning was not very clear before, and we had talked with Pete about it. Wanted to make it clear and they do no harm. Makes it clear for auto-scaling as well.